src/main/target/NAZE-rev6/target.h

```diff
@@ -0,0 +1,156 @@
+/*
```

Comment:
This shouldn't create a directory of its own, but instead a `NAZE_REV6.mk` file in the `NAZE` directory. (Also note the all caps and no hyphens.) This way it is possible to only define the differences, and not copy the whole definition, which gets rid of the extra effort for the additional target.